compiler/rustc_typeck/src/check/mod.rs

```diff
@@ -5777,7 +5777,9 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                         GenericParamDefKind::Lifetime => {
                             self.re_infer(Some(param), span).unwrap().into()
                         }
-                        GenericParamDefKind::Type { has_default, .. } => {
+                        // TODO should this return consts for defaults? Probably
```

Comment:
This file was moved, and is now `compiler/rustc_typeck/src/check/fn_ctxt.rs:2890`

I've left a placeholder, since I'm not quite sure how to properly handle this case.